_variables.yml

```
@@ -1,4 +1,6 @@
 status:
-  wip: '[{{< fa hammer size="xl" title="In progress" >}}]{style="color: #196440"}'
-  planned: '[{{< fa circle-pause size="xl" title="Planned" >}}]{style="color: #196440"}'
-  potential: '[{{< fa circle-question size="xl" title="Potential" >}}]{style="color: #196440"}'
+    done: '[{{< fa circle-check size="xl" title="Completed an MVP" >}}]{style="color: #196440"} *Completed*'
+    wip: '[{{< fa hammer size="xl" title="In progress" >}}]{style="color: #28a167"} *In progress*'
+    planned: '[{{< fa map size="xl" title="Planned" >}}]{style="color: #45d18e"} *Planned*'
+    potential: '[{{< fa circle-question size="xl" title="Potential" >}}]{style="color: lightgrey"} *Potential*'
+    ongoing: '[{{< fa repeat size="xl" title="Ongoing" >}}]{style="color: darkgrey"} *Ongoing*'
```

```suggestion
    ongoing: '[{{< fa repeat size="xl" title="Ongoing" >}}]{style="color: #196440"} *Ongoing*'
```

Ongoing have same colour as done.